tools/paquet/lib/paquet/rspec/tasks.rb

```diff
@@ -0,0 +1,147 @@
+# encoding: utf-8
+require "bundler"
+require "rake"
+require "rake/tasklib"
+require "fileutils"
+require "net/http"
+
+#
+# Uses bundler/fetcher to download stuff
+module Paquet
+  class Gem
+    RUBYGEMS_URI = "https://rubygems.org/downloads"
+
+    attr_reader :gems, :ignores
+
+    def initialize(target_path)
+      @target_path = target_path
+      @gems = []
+      @ignores = []
+    end
+
+    def add(name)
+      @gems << name
+    end
+
+    def ignore(name)
+      @ignores << name
+    end
+
+    def pack
+      FileUtils.rm_rf(@target_path)
```

Comment:
I don't currently have a logger in that tool for now and I would prefer not to use cabin to make sure we don't run into weird dependency problems with the package.

But I will provide feedback on that operation.